lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -6,7 +6,7 @@ class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmC
     expose :resize_confirm, :override_return => nil
     expose :resize_revert,  :override_return => nil
 
-    expose :validate_resize
+    expose :supports_resize?
```

Comment:
@gmcculloug I guess for backwards compatability we should still have `validate_resize` available?
can we do so by adding this?
```
def validate_resize
  {:available => supports_resize?, :message => unsupported_reason(:resize)}
end
```

in what context are those methods evaluated?